<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client_Window</class>
 <widget class="QMainWindow" name="Client_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client_Window</string>
  </property>
  <property name="state" stdset="0">
   <string>0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>431</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#InIPAddress[state = &quot;0&quot;]{
	
	background-color: rgb(255, 144, 146);
}
#InIPAddress[state = &quot;1&quot;]{
	
	background-color: rgb(153, 255, 170);
}</string>
    </property>
    <property name="title">
     <string>Connection Info</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="1">
      <widget class="QPushButton" name="btn_connect">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="InIPAddress">
       <property name="inputMask">
        <string>000.000.000.000;_</string>
       </property>
       <property name="text">
        <string>127.0.0.1</string>
       </property>
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
       <property name="state" stdset="0">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>IP Address</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinPort">
       <property name="minimum">
        <number>55004</number>
       </property>
       <property name="maximum">
        <number>65000</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="gr_send_data">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>431</width>
      <height>381</height>
     </rect>
    </property>
    <property name="title">
     <string>Client manager</string>
    </property>
    <widget class="QLineEdit" name="InData_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>341</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
     <property name="state" stdset="0">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>46</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Message</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_send_data_">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>341</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QListWidget" name="lst_Clients">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>130</y>
       <width>341</width>
       <height>241</height>
      </rect>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">overflow:scroll;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Client List</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>40</y>
      <width>351</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListWidget" name="lstConsole">
       <property name="styleSheet">
        <string notr="true">overflow:scroll;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clear_btn">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>clear_btn</sender>
   <signal>clicked()</signal>
   <receiver>lstConsole</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>542</x>
     <y>538</y>
    </hint>
    <hint type="destinationlabel">
     <x>566</x>
     <y>430</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
